### ISSUE TITLE
Alternative server with https and subdomain support

### ISSUE THREAD

**Issue by insign** [defunctzombie/localtunnel-server#131]
I created a permanent host to use, and implemented HTTPS support.

`lt -h "https://cloudkit.link" -p 80 --subdomain=whatever`

PR are welcome at https://github.com/cloudatlasid/tunnel-server

**Comment by bmkimagem** [comment]
@insign , I start tunnel-server in my cloud, but my client auto disconect. Can you help me?

**Comment by insign** [comment]
Hi, sure, you mean the local client or the server service?

**Comment by bmkimagem** [comment]
I installed the version of the tunnel server and apparently everything 
is ok. When I go to my client and direct to connect to my host, he 
connects but falls right away. And it on the server shows that you have 
a connection. I don't know why it's falling.



E Hélio escreveu:

**Comment by insign** [comment]
When you try to another server it works completely fine?

**Comment by bmkimagem** [comment]
Sim...quando eu conecto nesse seu https://cloudkit.link...funciona 
perfeito....
But I wanted to set up a server myself, as yours is working I wanted to 
know if you could help me to stabilize mine.

**Comment by insign** [comment]
Strange... Have you configured ssl?

**Comment by bmkimagem** [comment]
no, simple http ... ssl is mandatory for connection?

**Comment by bmkimagem** [comment]
On the server it informs that the client has connected, but in the api / 
status it informs 0 in the tunnel.
Do you have any tips on what it might be?

localtunnel:server making new client with id proud-dingo-86 +0ms


{"tunnels":0,"mem":{"rss":43745280,"heapTotal":10969088,"heapUsed":9099968,"external":1907554}}

**Comment by insign** [comment]
Have you disabled fw?

**Comment by bmkimagem** [comment]
That's right, I had to disable ... I was trying to add a rule ... Thank 
you

E2020-04-16 17:43, Hélio escreveu:

**Comment by insign** [comment]
You are welcome.

>

**Comment by iamadamreed** [comment]
@insign How are the dns subdomains handled in the server? Does the server have to host dns for the domain?

**Comment by cha-hin** [comment]
@insign 

Hello, I'm trying to accomplish the same as you on my own server. I generated my certificates and forwarded both ports `80` and `443` to `http://localhost:3000` on my server but I still get no output when I use `lt -h "https://myserver.something" -p xxxx`. Do you know where it could come from? 

Thanks a lot

**Comment by insign** [comment]
I'm sorry to say that I'm specialized in that project.

**Comment by cha-hin** [comment]
@insign 

It’s whatever I finally did it by myself.

**Comment by skiwithuge** [comment]
hi all! i have a question.. are you sure that the flow is full tls? by using wireshark on my server i see that some packets are in http

**Comment by KhanMaytok** [comment]
Any tutorial on this?
